pandas/tests/test_generic.py

```diff
@@ -734,7 +741,7 @@ def test_interpolate(self):
         non_ts[0] = np.NaN
         self.assertRaises(ValueError, non_ts.interpolate, method='time')
 
-    def test_interp_regression(self):
+    def test_interp_regression_pchip(self):
```

Comment:
just make this ``test_interp_pchip`` (and same below, no need for ``regression``)